OutOfSchool/OutOfSchool.IdentityServer/ViewModels/RegisterViewModel.cs

```diff
@@ -31,6 +31,10 @@ public class RegisterViewModel
     [EnumDataType(typeof(Gender), ErrorMessage = Constants.EnumErrorMessage)]
     public Gender Gender { get; set; }
 
+    [Required]
```

Comment:
need to think something about this, because the same viewmodel is used  for both parent & provider. And for provider this is not required as well as gender